### PR TITLE
Download and install CDM certificates from S3

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request modifies this Ansible role to download and install CDM public certificates from S3.  This involved switching from the [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role) skeleton to the [cisagov/skeleton-ansible-role-with-test-user](https://github.com/cisagov/skeleton-ansible-role-with-test-user) skeleton.

## 💭 Motivation and context ##

These public certificates are apparently sensitive data.

## 🧪 Testing ##

All `pre-commit` hooks and `molecule` tests pass.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

File: terraform/providers.tf [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# The provider used to create policies and roles that can read
+# parameters from AWS SSM Parameter Store in production.
+# provider "aws" {
+#   alias  = "images_production_ssm"
+#   region = var.aws_region
+#   assume_role {
+#     role_arn     = data.terraform_remote_state.images_production_ssm.outputs.provisionparameterstorereadroles_role.arn
+#     session_name = local.caller_user_name
+#   }
+# }
```
Comment (dav3r) [inline terraform/providers.tf]: Guessing that you meant to delete all of this?

File: terraform/remote_states.tf [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+#     encrypt        = true
+#     bucket         = "cisa-cool-terraform-state"
+#     dynamodb_table = "terraform-state-lock"
+#     profile        = "cool-terraform-readstate"
+#     region         = "us-east-1"
+#     key            = "cool-images-parameterstore/terraform.tfstate"
+#   }
+
+#   workspace = "production"
+# }
```
Comment (dav3r) [inline terraform/remote_states.tf]: Moar to delete?

**Review by felddy — APPROVED** [review]
redd fox disbelief

**Review by dav3r — APPROVED** [review]
These changes look good to me. 👍 
Note a couple of places where some commented-out lines can be deleted.

File: terraform/remote_states.tf (hunk 2 above)
Comment (jsf9k) [reply terraform/remote_states.tf]: See commit a3c4f3c.

File: terraform/providers.tf (hunk 1 above)
Comment (jsf9k) [reply terraform/providers.tf]: See commit a3c4f3c.

File: terraform/outputs.tf [hunk 3]
```diff
@@ -0,0 +1,30 @@
+output "access_key" {
+  value       = module.user.access_key
+  description = "The IAM access key associated with the CI IAM user created by this module."
```
Comment (hillaryj) [inline terraform/outputs.tf:3]: What does CI stand for in this case? I looked in the README - we should define the acronym somewhere, ideally where it's first used.

**Review by hillaryj — APPROVED** [review]
📜

File: terraform/outputs.tf (hunk 3 above)
Comment (hillaryj) [reply terraform/outputs.tf:3]: From usage elsewhere, it's likely the standard `continuous integration` and not something `certificate` related.

Comment (jsf9k) [reply terraform/outputs.tf:3]: In this context CI stands for continuous integration.  If we want to define the acronym then we should create a pull request in [cisagov/skeleton-ansible-role-with-test-user](https://github.com/cisagov/skeleton-ansible-role-with-test-user).

Comment (hillaryj) [reply terraform/outputs.tf:3]: Continuous Integration, per the skeletons. Resolving this since I was just confused by being in a repo about certs.

**Pull request merged**